.github/workflows/ci-test.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
         run: go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.12
       - name: Build Go
         run: |
-          make oasis-indexer
+          make nexus
```

Comment:
(no-op) I'd prefer `oasis-nexus` over `nexus` since we refer to it as Oasis Nexus in the docs. It'd also be consistent with Oasis Explorer and how most of our services historically were prefixed with `oasis-`. But since this rename is requested by marketing it's not up to us so I'll just lodge my objection here for posterity and we can leave as-is.

Reply:
I actually prefer `nexus` here because it's more of a token/identifier, and in that role, we never use `oasis-nexus` (with a hyphen); for consistency and simplicity, I like that we always have one or the other.
We mostly use "Oasis Nexus" in external docs when there is little external context and we want to additional qualify the name. We also use just "Nexus" in externally facing docs plenty of time.